fvm/context.go

```diff
@@ -93,6 +93,7 @@ func defaultContext(logger zerolog.Logger) Context {
 		ExtensiveTracing:              false,
 		TransactionProcessors: []TransactionProcessor{
 			NewTransactionVerifier(AccountKeyWeightThreshold),
+			NewTransactionServiceAccountChecker(),
```

Comment:
I think this should be part of the TransactionVerification, ideally we should avoid adding process for small operations, these simply adds a lot of unnecessary allocations.

Reply:
done [84ce8fb](https://github.com/onflow/flow-go/pull/2658/commits/84ce8fb676cb619a61af109098cbf789b5c5418f)

Reply:
hmm, in terms of code modularity, i actually liked the previous setup. I guess extra processors are currently not cheap to add?